docs/src/js/classes/VectorQuery.md

```diff
@@ -48,6 +48,37 @@ addQueryVector(vector): VectorQuery
 
 ***
 
+### analyzePlan()
+
+```ts
+analyzePlan(): Promise<string>
+```
+
+Executes the query and returns the runtime metrics.
+
+#### Returns
+
+`Promise`&lt;`string`&gt;
+
+execution plan with runtime metrics
+
+#### Example
+
+```ts
+import * as lancedb from "@lancedb/lancedb"
+const db = await lancedb.connect("./.lancedb");
+const table = await db.createTable("my_table", [
+  { vector: [1.1, 0.9], id: "1" },
+]);
+const plan = await table.query().nearestTo([0.5, 0.2]).analyzePlan();
```

Comment:
same comment applies in these other two places. I wonder if we could get docs automation to handle the inheritance for us at some point.

Reply:
Those are auto generated docs, let me see fix the original place